test/customRegistries.test.ts

```diff
@@ -63,8 +63,7 @@ suite("Custom registries", async function (this: Suite): Promise<void> {
         test("Connect, no auth", async function (this: Context) {
             let input = new TestUserInput([
                 'http://localhost:5900',
-                'fake username', // TODO: TestUserInput doesn't currently allow '' as an input
-                'fake password'
+                ''
```

Comment:
Password?

Reply:
We don't ask for a password if there's no username.